python/GafferSceneUI/_InspectorColumn.py

```diff
@@ -517,6 +517,9 @@ def __dragEnter( column, path, pathListing, event ) :
 	if path is None :
 		return False
 
+	if not isinstance( event.data, IECore.Data ) :
```

Comment:
Is there an argument for accepting `IECore.Object`, since that's what `Inspector::canEdit()` and `Inspector::edit()` work with?

Reply:
Fair point! No need to be quite so restrictive. I've switched over to `IECore.Object` in f1133be.